src/rewards/rewards.utils.ts

```diff
@@ -255,3 +257,42 @@ export async function getProtocolValueSources(
     return [];
   }
 }
+
+export function getRewordLoggerInst(chain: Chain): RewardsLoggerInst | undefined {
+  let rewardsLogger: RewardsLoggerInst | undefined;
+
+  if (chain.rewardsLogger) {
+    rewardsLogger = RewardsLogger__factory.connect(chain.rewardsLogger, chain.batchProvider);
+  }
+
+  return rewardsLogger;
+}
+
+export async function getRewardSchedules(
```

Comment:
https://github.com/Badger-Finance/badger-sdk/blob/793a0b1c3492646cae66e35f39198d6b5a3be292/src/rewards/rewards.service.ts#L52-L77

should we use this instead and we can build on top of this?

Reply:
if we want to utilize sdk in api, i think we can

Reply:
ohh ye, there is a problem, sdk doesnt expose `duration` to count complition %

Reply:
maybe can we make a pr on that one? and we could actually just expose the completion on there too i think its a useful metric to serve

Reply:
ye sure, will do the ticket